heron/stmgr/src/cpp/manager/stmgr-server.cpp

```diff
@@ -420,8 +421,8 @@ void StMgrServer::SendToInstance2(sp_int32 _task_id,
                                   const proto::system::HeronTupleSet2& _message) {
   TaskIdInstanceDataMap::iterator iter = instance_info_.find(_task_id);
   if (iter == instance_info_.end() || iter->second->conn_ == NULL) {
-    LOG(ERROR) << "task_id " << _task_id << " has not yet connected to us. Dropping..."
-               << std::endl;
+    LOG(ERROR) << "task_id " << _task_id << " has not yet connected to us. Dropping...";
+    stmgr_server_metrics_->scope(METRIC_BYTES_TO_INSTANCES_LOST)->incr_by(_message.ByteSize());
```

Comment:
I did not find the CachedByteSize() in protobuf.
There is SpaceUsed() which is noticeably slower than ByteSize()